packages/libs/error-stack/src/compat/mod.rs

```diff
@@ -0,0 +1,23 @@
+#[cfg(feature = "anyhow")]
+mod anyhow;
+#[cfg(feature = "eyre")]
+mod eyre;
+
+/// Compatibility trait to convert from external libraries to [`Report`].
```

Comment:
Do you want to quickly explain here _why_ we give this and why we can't use IntoReport